packages/orbit-components/src/Tooltip/__tests__/index.test.js

```diff
@@ -1,13 +1,21 @@
 // @flow
 import * as React from "react";
-import { shallow } from "enzyme";
+import { mount } from "enzyme";
 
 import Tooltip from "../index";
 import Airplane from "../../icons/Airplane";
 
+jest.mock("../../hooks/useMediaQuery", () => {
+  return () => {
+    return {
+      isLargeMobile: true,
+    };
+  };
+});
+
```

Comment:
Nice one 👍

Reply:
Do you have any idea how I could re-mock that for a different test? Eg I want to test for `isLargeMobile: true` and also for `false`. I was not able to do that simply 🤔

Reply:
[`jest.doMock`](https://jestjs.io/docs/en/jest-object#jestdomockmodulename-factory-options) is for mocking per-test. Is that what you meant? But I'm not sure if `require` will work here, if it doesn't try appending `.default` to the result of `require` call.